src/main/java/org/isf/dicom/model/FileDicom.java

```diff
@@ -260,15 +257,10 @@ public void setDicomData(Blob dicomData) {
 	 *            the dicomFile to set
 	 */
 	public void setDicomData(File dicomFile) {
-		try {
-
-			FileInputStream fis = new FileInputStream(dicomFile);
+		try (FileInputStream fis = new FileInputStream(dicomFile)) {
 			byte[] byteArray = new byte[fis.available()];
 			fis.read(byteArray);
-			fis.close();
```

Comment:
should we put the FileInputStream.close() in the a finally block? it should be needed to release the resource

Reply:
This is using `try-with-resources` which automatically handles the closing of the resource.

Reply:
correct! my bad